release/images.sh

```diff
@@ -59,7 +59,7 @@ finalize_image() {
     # ${3} - List of image files to be finalized
     declare -n _output=${1}
     local name=${2}
-    local file_list=("${3}")
+    local file_list=(${3})
```

Comment:
on line 71 transform the array on a string again. Maybe just remove the usage of an array there and change line 71 to not be within quotes to allow the word splitting?

Reply:
```
$ string="a b c"
$ for a in "$string"; do echo "--"; echo $a; done
--
a b c
$ for a in $string; do echo "--"; echo $a; done
--
a
--
b
--
c
```

Reply:
done

Reply:
@mbelluzzo shell check doesn't seem to be happy with this exception
https://github.com/koalaman/shellcheck/wiki/SC2128

Reply:
You will have to add an exception in the for, but you are also missing change the attribution to be `local file_list="${3}", is still an array.

Reply:
done